templates/block/block--views-block--news-latest-news-block.html.twig

```diff
@@ -0,0 +1,43 @@
+{#
+/**
+ * @file
+ * Theme override to display a block.
+ *
+ * Available variables:
+ * - plugin_id: The ID of the block implementation.
+ * - label: The configured label of the block if visible.
+ * - configuration: A list of the block's configuration values.
+ *   - label: The configured label for the block.
+ *   - label_display: The display settings for the label.
+ *   - provider: The module or other provider that provided this block plugin.
+ *   - Block plugin specific settings will also be stored here.
+ * - content: The content of this block.
+ * - attributes: array of HTML attributes populated by modules, intended to
+ *   be added to the main container tag of this template.
+ *   - id: A valid HTML ID and guaranteed unique.
+ * - title_attributes: Same as attributes, except applied to the main title
+ *   tag that appears in the template.
+ * - title_prefix: Additional output populated by modules, intended to be
+ *   displayed in front of the main title tag that appears in the template.
+ * - title_suffix: Additional output populated by modules, intended to be
+ *   displayed after the main title tag that appears in the template.
+ *
+ * @see template_preprocess_block()
+ */
+#}
+{%
+  set classes = [
+  'sub-menu',
+  'rel-menu',
+]
+%}
+<article{{ attributes.removeClass('views-element-container').addClass(classes).setAttribute('aria-labelledby', 'latest-news-block').removeAttribute('id') }}>
```

Comment:
I'd be tempted to move this up below set classes and tweak the values to make it more readable.

Reply:
tidied by setting all the attributes in a logical sequence at the top of the template